src/sentry/migrations/0726_backfill_groupsearchviews_with_pinned_searches.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 5.0.6 on 2024-06-03 17:27
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_groupsearchviews_with_pinned_searches(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    GroupSearchView = apps.get_model("sentry", "GroupSearchView")
+    SavedSearch = apps.get_model("sentry", "SavedSearch")
+
+    pinned_searches = SavedSearch.objects.filter(visibility="owner_pinned")
+    for pinned_search in RangeQuerySetWrapperWithProgressBar(pinned_searches):
+        GroupSearchView.objects.update_or_create(
+            organization=pinned_search.organization,
+            user_id=pinned_search.owner_id,
+            position=0,
+            defaults={
+                "name": "Default Search",
+                "query": pinned_search.query,
+                "query_sort": pinned_search.sort,
+            },
+        )
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
```

Comment:
```suggestion
    is_post_deployment = True
```

Databackfills should be run post-deploy. We only have 5s of migration runtime during deploys.

Reply:
Makes sense, thanks!